src/encoded/static/components/summary.js

```diff
@@ -345,45 +326,51 @@ SummaryData.defaultProps = {
 class SummaryBody extends React.Component {
     constructor(props) {
         super(props);
+        this.state = {
+            CellCount: 0
```

Comment:
variables should start with a lower case letter... identifiers starting with upper case are used for classes and constants only

Reply:
updated